soc/arm/ambiq/apollo4x/soc.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2023 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SOC_H__
+
+#include <am_mcu_apollo.h>
+#include <am_bsp.h>
+
+#define __SOC_H__
```

Comment:
wrong header guard. why is bsp needed here?

Reply:
Fixed the header guard. BSP include is not needed here, I've removed it.